Assets/MRTK/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -187,8 +187,8 @@ Shader "Mixed Reality Toolkit/Standard"
             #include "UnityStandardUtils.cginc"
             #include "MixedRealityShaderUtils.cginc"
 
-            // This define will get commented in by the UpgradeShaderForLightweightRenderPipeline method.
-            //#define _LIGHTWEIGHT_RENDER_PIPELINE
+            // This define will get commented in by the UpgradeShaderForUniversalRenderPipeline method.
+            #define _RENDER_PIPELINE
```

Comment:
Could you re-comment out this line so that the shader works with the built-in render pipeline by default?

```suggestion
            //#define _RENDER_PIPELINE
```